### ISSUE TITLE
Document good examples on using Lucene style syntax for searching

### ISSUE THREAD

**Issue by mdavoodi** [mohamicorp/search-for-stash-issues#35]

**Comment by mdavoodi** [comment]
Here's a comment from one of our customers:

```
    For example, while I have some time to dig out the index and figure out how to do a query that I want, my developers do not.   Let me give you an example or a search that I performed:
 
"<artifactId>cbn-cryptography</artifactId>" path:pom.xml repository:*-cbn-*
 
This worked quite well, but I had to dig out the index to figure out what keywords to use.
```

**Comment by mdavoodi** [comment]
Also add the smart query monikers to search. Look at how Atlassian default search shows its examples.

**Comment by mdavoodi** [comment]
This has now been implemented in [version 2.7.2](https://marketplace.atlassian.com/plugins/com.mohamicorp.stash.search-for-stash/server/overview).
